common/libs/contract/ABIFunction.ts

```diff
@@ -0,0 +1,188 @@
+// @flow
+import abi from 'ethereumjs-abi';
+import { toChecksumAddress } from 'ethereumjs-util';
+import BigNumber from 'bignumber.js';
+import { FuncParams, OutputMappings, Output, Input } from './types';
+export default class AbiFunction {
```

Comment:
More nitpicks :) - can we enforce newlines between class declarations and imports?